src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -39,7 +39,7 @@ public static List<PartyDetails> buildHearingPartiesDetails(HearingWrapper wrapp
         }
 
         if (isYes(caseData.getJointParty().getHasJointParty())) {
-            partiesDetails.add(createJointPartyDetails(caseData));
+            partiesDetails.addAll(buildHearingPartiesPartyDetails(caseData.getJointParty(),null,null,null,null,appellant.getId()));
```

Comment:
Cleaner to add an extra buildHearingPartiesPartyDetails method without the values where nulls have been passed in.  E.g.:
public static List<PartyDetails> buildHearingPartiesPartyDetails(Party party, String appellantId) {
    return buildHearingPartiesPartyDetails(party,null,null,null,null,appellantId);
}

Reply:
Done